api/swagger.yaml

```diff
@@ -864,7 +864,7 @@ definitions:
       Healthcheck:
         $ref: "#/definitions/HealthConfig"
       ArgsEscaped:
-        description: "Command is already escaped (Windows only)"
+        description: "Command is already escaped (Windows only). Not used and always ignored."
```

Comment:
I know; too early for review 🤗 - I guess we should "deprecate" this in API v1.41 (older versions of the API, at least when running on an older daemon) would still need it